model/utils.js

```diff
@@ -235,9 +235,9 @@ THE FOLLOWING FUNCTIONS IMPLEMENT THE DIFFERENT VARIATIONS OF THE CRUD OPERATION
 async function retrieve(sql, params = [], message = defaultMsg) {
     console.log(
         "-- The following query is being executed --\n sql: " +
-            sql +
```

Comment:
Try to avoid modifying files in a PR unless that is the objective.

Reply:
wasn't intentional boss 🥲